service/discovery/azure/compute.go

```diff
@@ -154,7 +154,7 @@ func (*azureComputeDiscovery) handleFunction(function *armappservice.Site) voc.I
 				},
 				Labels: labels(function.Tags),
 			},
-			NetworkInterface: []voc.ResourceID{},
+			NetworkInterfaces: []voc.ResourceID{},
```

Comment:
Do we have to adjust any Rego rules? I think the related-resources stuff is not yet merged in, so we definitely need to keep this in mind once we do.

Reply:
What should I do with that. We also could do it in singular. I can live well with both :woman_shrugging:

Reply:
I think in general it should be plural, the only issue is that `storage` has no plural :)